packages/dev/core/src/Rendering/GaussianSplatting/gaussianSplatting.ts

```diff
@@ -376,4 +442,18 @@ export class GaussianSplatting {
         this.mesh?.dispose();
         this.mesh = null;
     }
+
+    private _getTextureSize(length: number): Vector2 {
+        const maxTextureSize = this.scene.getEngine().getCaps().maxTextureSize;
+        const width = maxTextureSize;
+        let height = 1;
+        while (width * height < length) {
+            height *= 2;

```

Comment:
In anything but WebGL1, you can do `height++` to tighten the size of the texture as much as possible, as the texture can be quite big.
You can check for WebGL1 by doing `engine.webGLVersion === 1`.

Reply:
Done! I have added an additional condition for WebGPU, as WebGPU will also fall into the webGLVersion 1.
I appreciate all of your comments, they are truly helpful for my code.